pages/deposit/[symbol].tsx

```diff
@@ -39,8 +43,15 @@ export const getServerSideProps: GetServerSideProps<{
 const Deposit: NextPage<
   InferGetServerSidePropsType<typeof getServerSideProps>
 > = ({ asset }) => {
+  const { balances } = useCurrencyContext()
+  const { deposit } = useDepositContext()
   const [selected, _setSelected] = useState(0)
```

Comment:
How about change name to `selectedEpoch`?

Reply:
I'll change it to `selectedEpochs`